homeassistant/components/tilt_pi/entity.py

```diff
@@ -0,0 +1,39 @@
+"""Base entity for Tilt Pi integration."""
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import TiltPiDataUpdateCoordinator
+from .model import TiltHydrometerData
+
+
+class TiltEntity(CoordinatorEntity[TiltPiDataUpdateCoordinator]):
+    """Base class for Tilt entities."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: TiltPiDataUpdateCoordinator,
+        hydrometer: TiltHydrometerData,
+    ) -> None:
+        """Initialize the entity."""
+        super().__init__(coordinator)
+        self._mac_id = hydrometer.mac_id
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, hydrometer.mac_id)},
```

Comment:
what's a mac_id? Is it a mac? is it an ID?

Reply:
The `mac_id` represents this identifier in controllers: https://en.wikipedia.org/wiki/MAC_address

`mac_id` matches what the API returns and feels natural to me, but maybe `mac_address` is less ambigous? In any way, it is an identifier, and I say this understanding that "id" is an extremely overloaded word.

Open to going either way.